search-solr/src/main/java/edu/unc/lib/boxc/search/solr/models/ContentObjectSolrRecord.java

```diff
@@ -71,19 +77,19 @@ public String getIdWithoutPrefix() {
     }
 
     @Override
-    public CutoffFacet getAncestorPathFacet() {
+    public CutoffFacetImpl getAncestorPathFacet() {
         return ancestorPathFacet;
     }
 
-    public void setAncestorPathFacet(CutoffFacet ancestorPathFacet) {
+    public void setAncestorPathFacet(CutoffFacetImpl ancestorPathFacet) {
         this.ancestorPathFacet = ancestorPathFacet;
     }
 
     @Override
     @Field
     public void setAncestorPath(List<String> ancestorPaths) {
         super.setAncestorPath(ancestorPaths);
-        this.ancestorPathFacet = new CutoffFacet(SearchFieldKeys.ANCESTOR_PATH.name(), ancestorPaths, 0);
+        this.ancestorPathFacet = new CutoffFacetImpl(SearchFieldKey.ANCESTOR_PATH.name(), ancestorPaths, 0);
```

Comment:
This may be a dumb question, but what exactly is a cutoff facet or value?

Reply:
The solr code was poorly documented before (which is my fault). I've gone ahead and added longer descriptions of cut off facet classes and fixed a comment:
https://github.com/UNC-Libraries/Carolina-Digital-Repository/pull/1283/commits/01d266e718dc904552e1cf10f0e6b0ae4f8145dd